app/views/admin/permissions.html.erb

```diff
@@ -33,6 +34,7 @@
           </label>
         </td>
       <% end %>
+      <td><%= link_to "Nuke", 'javascript:void(0);', class: 'text-danger nuke-user-link', data: { :user_id => user.id, :username => user.username } %></td>
```

Comment:
Can we wrap this in a `<% if current_user&.has_role?(:developer) %>` to restrict it down to developers for now please?

Reply:
[status-completed]; `verify_developer` added to controller as well.